### PR TITLE
docs: prepare 2.16.5 changelog

### PR DESCRIPTION
**Pull request by mikehardy**
I really struggled with how to explain the analytics lapse

My goal was to accurately describe

- the actual change
- how we treat the data in general and currently (anonymized + currently disabled because of GA4 porting needs)
- our regret at the error vs our standards

I'm open to any changes on any of it as long as they generally satisfy the goals above and sound like a human vs some person reading or writing a fake apology from a script

### CODE REVIEW

**Comment by mikehardy** [comment]
related thought - just putting this here before I forget about it - perhaps we should rename the analytics opt-in key in a future 2.17 release, and make it such that the new key is checked for opt-in-ness so this old key which now perhaps has some corrupt data on device from incorrect defaults is ignored? Basically just wipe the opt-in-ness clean.

Our decision-making on analytics is based on percentages of devices reporting, so this would radically lower the number of devices reporting, to start, but I do not believe it would affect any decisions we would make about feature usage in the future, and it would demonstrate a commitment to opt-in by never using the potentially corrupt setting 🤔

**Comment by mikehardy** [comment]
> I believe (needs confirmation) this was only triggered if the user opened their settings AND a default from an old version hadn't been set

I think you're right, but if it's possible - even a tiny crack of possible - it's a trust issue, and if we *can* make it a zero percent chance, I'd like it...

**Comment by david-allison** [comment]
I'd support renaming the key + opting everyone out by default

**Comment by david-allison** [comment]
My note was only that the unintentional opt-in definitely WILL have occurred and it shouldn't, but the impact was more limited than 'analytics is opt-out by default'

**Review by david-allison — CHANGES_REQUESTED** [review]
Proposal (ignorable): statement here that we'll follow up in a further update to opt everyone out as a precaution

As this is time sensitive, I'd rather not hastily make a change here, but it shows our commitment and guarantees a follow-through on our behalf

File: changelog.asc [hunk 1]
```diff
@@ -2,6 +2,16 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.16.5 (20230906)
+* Fix potential crash in our crash report system. See: Murphy's Law
+* Fix incorrect default setting for analytics opt-in. Should be default off.
+** Please check your setting if you want to make sure it is off
+** Note1: our analytics is always anonymized and never shared as a first step
+** Note2: the backend has been disabled for months, so there should be no exposure
+** Still this was in error and counter to our strict opt-in ethos. We are deeply sorry.
```
Comment (mikehardy) [inline changelog.asc:11]:
```suggestion
** Still this was in error and counter to our strict opt-in ethos. We are deeply sorry. 
** We will issue a future update shortly to opt everyone out as a precaution
```

**Review by david-allison — APPROVED** [review]

**Pull request merged**